index.html

```diff
@@ -907,15 +937,23 @@ <h2>Terminology</h2>
   <dfn>Servient</dfn>,
   <dfn>WoT Interface</dfn>,
   <dfn>WoT Runtime</dfn>,
-  etc. is defined in <a href="https://w3c.github.io/wot-architecture/#terminology">section 3</a>
-  of the WoT Architecture document [[WOT-ARCHITECTURE]].
+  etc. is defined in <a href="https://w3c.github.io/wot-architecture/#terminology">Section 3</a>
+  of the WoT Architecture specification [[WOT-ARCHITECTURE]].
   </p>
 
   <p>
       In addition, this document introduces the following definitions:
```

Comment:
I see the change from "document" to "specification".   Should this be changed too?
Note: Editorial.  This can change can be deferred, and personally I think either is fine in this case, but it would be good to be consistent.

Reply:
CHanged to specification